openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -39,12 +39,17 @@ module Api
                        'cmd_raw_no_range_check',
                        'cmd_raw_no_hazardous_check',
                        'cmd_raw_no_checks',
-                       'build_command',
+                       'build_cmd',
+                       'build_command', # DEPRECATED
                        'send_raw',
-                       'get_all_commands',
-                       'get_all_command_names',
-                       'get_command',
-                       'get_parameter',
+                       'get_all_cmds',
+                       'get_all_commands', # DEPRECATED
+                       'get_all_cmd_names',
+                       'get_all_command_names', # DEPRECATED
+                       'get_cmd',
+                       'get_command', # DEPRECATED
+                       'get_param',
+                       'get_parameter', # DEPRECATED
```

Comment:
I wanted to make our APIs consistent with the use of `cmd` vs `command`. They all use `cmd` now.